### ISSUE TITLE
Traceback (most recent call last):   File "trape.py", line 23, in <module>     from core.utils import utils   File "/home/kali/Desktop/trape/core/utils.py", line 21, in <module> import requests, json ImportError: No module named requests

### ISSUE THREAD

**Issue by MrRat14** [jofpin/trape#249]
Traceback (most recent call last):
  File "trape.py", line 23, in <module>
    from core.utils import utils                  #                        
  File "/home/sojib/Desktop/trape/core/utils.py", line 21, in <module>     
    import requests, json                                                  
ImportError: No module named requests

Note: How to slove this problem please someone help me

**Comment by JoshuaPoddoku** [comment]
`  File "C:\Users\joshu\Desktop\Trape\trape\core\utils.py", line 24, in <module>
    import httplib
ModuleNotFoundError: No module named 'httplib'`

Facing similar error

**Comment by v8ive** [comment]
TRIAL 1:

$ python trape.py -h
Traceback (most recent call last):
  File "trape.py", line 23, in <module>
    from core.utils import utils                  #
  File "/home/v8ive/trape/core/utils.py", line 21, in <module>
    import requests, json
ImportError: No module named requests

TRIAL 2:

$ python2 trape.py -h
Traceback (most recent call last):
  File "trape.py", line 23, in <module>
    from core.utils import utils                  #
  File "/home/v8ive/trape/core/utils.py", line 21, in <module>
    import requests, json
ImportError: No module named requests

TRIAL 3:

python3 trape.py -h
Traceback (most recent call last):
  File "trape.py", line 23, in <module>
    from core.utils import utils                  #
  File "/home/v8ive/trape/core/utils.py", line 23, in <module>
    import httplib
ModuleNotFoundError: No module named 'httplib'

-----------------------------------------------------------------------------------------------

In conclusion, I have no idea what is wrong. I understand that Python3 renamed "httplib" to "http.client" but changing that in "~/trape/core/utils.py" (as it references in the error where it tries to import "httplib") resulted in this:

$ python3 trape.py -h
Traceback (most recent call last):
  File "trape.py", line 24, in <module>
    from core.trape import Trape                  #
  File "/home/v8ive/trape/core/trape.py", line 16, in <module>
    from core.dependence import urllib2
  File "/home/v8ive/trape/core/dependence/urllib2.py", line 225
    raise AttributeError, attr
                        ^
SyntaxError: invalid syntax

**Comment by JoshuaPoddoku** [comment]
Let's ask if we can mark as an issue and make this happen
